go-controller/pkg/ovn/ovn.go

```diff
@@ -10,13 +10,15 @@ import (
 	"sync"
 	"time"
 
+	ctypes "github.com/containernetworking/cni/pkg/types"
```

Comment:
this import group name is very confusing to me; I would expect this one to be `cnitypes` ...

which would make the "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/cni/types" one, possibly `ovncnitypes` ? 

What are we using for these import groups in the project ? As long as we're consistent throughout the project, I think everything is fine.

Reply:
ok